uiowa/uiowa
# Feature: Academic groups

This resolves https://github.com/uiowa/uiowa/issues/3110 and resolves https://github.com/uiowa/uiowa/issues/3130. 

This pr does not contain "Have ability to filter by these groups on future Areas of study search page", that should be addressed in https://github.com/uiowa/uiowa/issues/3203. 

# How to test

- `blt ds --site=admissions.uiowa.edu`
- `yarn workspace uids_base gulp --development`
- `drush @admissions.local uli`
- Add a "Business, Economics, and Management" taxonomy term to "Academic Groups" at: http://admissions.local.drupal.uiowa.edu/admin/structure/taxonomy/manage/academic_groups/add and add Name, Description, Add an image, and a Video. 
<details><img width="1372" alt="Screen Shot 2021-03-08 at 11 20 44 AM" src="https://user-images.githubusercontent.com/1036433/110358728-53d9ba80-8002-11eb-88f4-a02c0692d935.png"></details>

- Add a reference to "Business, Economics, and Management" on an Area of Study page: http://admissions.local.drupal.uiowa.edu/node/1021/edit?destination=/admin/content and verify that the page is referenced at the bottom of http://admissions.local.drupal.uiowa.edu/academic-groups/business-economics-and-management in a 3 column card grid. 
- Go to https://admissions.local.drupal.uiowa.edu/academics/groups and place the "Academic Groups Block" onto the layout builder page and verify that it renders as a three column card grid:

<details><img width="1332" alt="Screen Shot 2021-03-08 at 11 21 04 AM" src="https://user-images.githubusercontent.com/1036433/110358633-40c6ea80-8002-11eb-9a6c-a89c386ed475.png"></details>

- Masquerade as a `webmaster`, `editor`, or `publisher` and verify that you can edit the Academic Groups taxonomy.
